packages/tocco-ui-showcase/src/toccoUtilExamples/notifier/example.js

```diff
@@ -77,21 +78,24 @@ class Example extends React.Component {
     this.store.dispatch(notifier.modalComponent(
       id,
       'Title',
-      'Please wait',
+      'Message',
       props => (
-        <div style={{border: '1px dotted red'}}>
-          <p>My Custom-Component</p>
-          <Button
-            ink="primary"
-            label="primary action"
-            onClick={props.close}
-          />
-          <Button
-            label="secondary action"
-            onClick={props.close}
-          />
-        </div>
-      )
+        <React.Fragment>
```

Comment:
Have you tried if  the Fragment shorthand works in our setup:

    <>
      <ChildA />
    </>

Reply:
So far I know that babel-loader throws a SyntaxError and Sublime's syntax highlighting fails.